### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by 405421622** [Chia-Network/chia-blockchain#8065]
Bug's wallet appears to be functioning normally. There is synchronization and there is preliminary screening. But the farm inside the Full Node Connection Kib upload is always 0. Download data is also available. Also asked around a lot of people have the same mistake. I hope it's official

**Comment by xxxguoxinxxx** [comment]
相同的问题，爆块减少了

**Comment by 405421622** [comment]
不是爆块减少.是没有了爆块了.只有矿池能爆块.

**Comment by 405421622** [comment]
![QQ图片20210814203957](https://user-images.githubusercontent.com/88928589/129446630-cc4c3067-d74b-4367-9ebb-3fe43862af49.png)

**Comment by xxxguoxinxxx** [comment]
![450dd14af69e6cec44671d8ef4686eb](https://user-images.githubusercontent.com/49920095/129446706-2e686048-d6d6-4f3e-a969-94b1c6cf0b1c.png)

**Comment by xxxguoxinxxx** [comment]
我这还要爆块的，但是爆块比之前少很多。

**Comment by 405421622** [comment]
基本不爆，没有上传数据

**Comment by 405421622** [comment]
爆也是爆给矿池

**Comment by xxxguoxinxxx** [comment]
该如何是好啊

**Comment by xxxguoxinxxx** [comment]
这跟有没有公网ip，没有关系吧！

**Comment by 405421622** [comment]
目前很多矿工都有这个问题.多发点帖子让更多的关注

**Comment by 405421622** [comment]
很多人没注意以为只是算力大了，不爆块

**Comment by xxxguoxinxxx** [comment]
1.2.0有这个问题吗？还能退回去不？

**Comment by 405421622** [comment]
不清楚.不过可以试试

**Comment by lumiamax** [comment]
我的也是这样，不清楚以前是否有上传，但是差不多3pib，接矿池有爆块，solo已经15天以上不爆块了

**Comment by tracy32792** [comment]
> 我的也是这样，以前认识的有上传，还有3pib，接矿池有爆块，solo已经15天以上不爆块了

me too

**Comment by xxxguoxinxxx** [comment]
有没有大神，解答一下！

**Comment by tracy32792** [comment]
maybe some people in DIScard  can solve this error

**Comment by xxxguoxinxxx** [comment]
有DIScard的链接吗？

**Comment by xxxguoxinxxx** [comment]
没得人关注了？

**Comment by loppefaaret** [comment]
> Bug's wallet appears to be functioning normally. There is synchronization and there is preliminary screening. But the farm inside the Full Node Connection Kib upload is always 0. Download data is also available. Also asked around a lot of people have the same mistake. I hope it's official

If you are farming on a pool, using the new plotNFT format, there is very little traffic from the farmer -> to the full node - practically only once a block is bursted / exploded / won. and even then, the traffic is minimal, and will take time to show up on a KiB scale.

To other users in this issue, with other problems, i suggest you either find an already opened issue that is the same as yours, or create a new issue.

**Comment by 405421622** [comment]
> > Bug 的钱包似乎运行正常。有同步，有可能出现。但是全节点连接 Kib 上传的农场总是0。下载数据也可以的。也问过很多人正常的错误。希望是官方的
> 
> 如果你在一个池子上耕种，使用新的plotNFT格式，从农民- >到完整节点的流量很少-实际上只有当一个块被爆破/爆炸/时获胜即便如此，流量也是最小的，需要时间才能达到 KiB 规模。
> 
> 对于此问题的其他用户，如果有其他问题，我建议您找到与您相同的已打开问题，则不会创建一个新问题。

都是老协议、具体怎么弄好他.说一下.别人算力小没察觉，但是算力3P一个月不爆块不就是出问题了吗/ 谢谢

**Comment by shaneo257** [comment]
If you look at the Advanced option under your farmer tab, Your Full Node Connection will match to what is showing in your Full Node Tab, under connections
![image](https://user-images.githubusercontent.com/36589806/129689893-4d8c857e-5dc2-4d2e-83f7-73a4662f5eaa.png)
![image](https://user-images.githubusercontent.com/36589806/129690031-763f134a-29a5-489e-95da-ae11552e10bd.png)

There is no bug apart from the farm tab shows it in KiB Up/Down, whilst Full Node shows it as MiB Up/Down

**Comment by xxxguoxinxxx** [comment]
我就想知道怎么获取奖励XCH变得少了很多。

**Comment by shaneo257** [comment]
well your currently fighting with an increasing netspace and also luck has allot to do with your block wins as well

**Comment by xxxguoxinxxx** [comment]
这好像根运气没有关系，这个月获得奖励就是不正常
